### DISCUSSION TITLE
slurm-web / RackDB - API server error (400): Unable to find infrastructure localcluster in database

### DISCUSSION THREAD

**Discussion by stadox** [rackslab/Slurm-web#394] [Q&A]
Hello,

I'm trying to figure out what is the cause of this error message shown on the screen:

![screen](https://github.com/user-attachments/assets/3f6527f9-2c1b-4228-8142-6fa35e925fa1)

![console](https://github.com/user-attachments/assets/21cdf334-c94f-4b3c-9277-5429ef6c4ab8)

I went through the guide:
https://docs.rackslab.io/slurm-web/install/quickstart.html 
https://docs.rackslab.io/racksdb/install/quickstart.html

installed RacksDB
configure RacksDB to my local environment

If I do start racksdb-web --with-ui 
I can browse to the web page http://localhost:5000/ and see the environment I have configured.

I'm using default tags=compute
https://docs.rackslab.io/slurm-web/conf/conf/agent.html#_racksdb

May I ask you for any hint or an advice what to check?

Thank you in advance

Regards,
S.

**Comment by rezib** [comment]
What is the output of `racksdb infrastructures --list` on your host?

**Reply by stadox** [reply]
racksdb infrastructures --list
localcluster

**Reply by rezib** [reply]
Can you reproduce the error with the following settings in `/etc/slurm-web/agent.ini`?

```
[service]
debug=yes
debug_flags=
  slurmweb
  racksdb
```

This should give a hint about what is really going on in the agent service logs.

**Reply by stadox** [reply]
Hello,

weird, now it shows the rack + server.

What I did was only to amend the [service] with your data and restarted the slurm-web-agent-uwsgi.service and reload the web page.

Weird because I did the same on Friday.

Thank you for time a support. Now the system seems to be fully working.

Regards,
S.
